### PR TITLE
[14.0][FIX] hr_expense_invoice: do not display UoM and quantity label

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Hide quantity label and UoM field when selecting an invoice, as it is done with quantity field.

Before:
Peek 2022-05-25 15-48

After:
Peek 2022-05-25 15-48-2

@ForgeFlow

### CODE REVIEW

File: hr_expense_invoice/models/hr_expense.py [hunk 1]
```diff
@@ -95,6 +95,7 @@ def _onchange_invoice_id(self):
         """
         if self.invoice_id:
             self.quantity = 1
+            self.product_uom_id = self.product_id.uom_id
```
Comment (pedrobaeza) [inline hr_expense_invoice/models/hr_expense.py:98]: Why is this needed?

**Review by mariadforgeflow — APPROVED** [review]

File: hr_expense_invoice/models/hr_expense.py (hunk 1 above)
Comment (LoisRForgeFlow) [reply hr_expense_invoice/models/hr_expense.py:98]: To ensure that it is set, maybe the user removed it after setting the product.

Comment (pedrobaeza) [reply hr_expense_invoice/models/hr_expense.py:98]: OK, a very specific case, but let's cover it.

**Review by pedrobaeza — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
@pedrobaeza merge?

**Comment by pedrobaeza** [comment]
Please fw-port it to v15
